### ISSUE TITLE
Version 1.5.6 is raising DAR201 for functions with a bare "return" in them

### ISSUE THREAD

**Issue by DrewDevereux** [terrencepreilly/darglint#141]
Version 1.5.6 is raising `DAR201 Missing "Returns" in Docstring` for functions with a bare `return` in them. I think this is a mistake. If not, then at least it should raise a separate error code so that there is the option of disabling it independently of other DAR201 errors.

**Comment by terrencepreilly** [comment]
Yes, this was a mistake.  Should be fixed in 4b5bc3342e2537101233b5f7a9ecb99c8b46d1e9, and will be included in v1.5.8.

**Comment by fredrikaverpil** [comment]
Hi again @terrencepreilly 

What about this scenario?
With darglint 1.5.8, I still get "excessive exceptions in Raises section" (DAR402) for both HTTPError and ConnectionError

```python
"""ddd."""

import requests


def something():
    """Something."""
    pass


def test_something():
    """Test.

    Raises:
        HTTPError: if call to something fails...
        ConnectionError: if call to something fails...

    """
    try:
        something()
    except (requests.HTTPError, requests.ConnectionError) as err:
        print(err)
        raise
```
